libcontainer/specconv/example.go

```diff
@@ -155,10 +158,66 @@ func Example() *specs.Spec {
 	}
 }
 
+// RootlessOpts is an optional spec for ToRootless
+type RootlessOpts struct {
+	// Add all subuids/subgids to spec.Linux.{U,G}IDMappings.
+	// Note that in many cases users shouldn't be mapping all of their
+	// allocated subuids/subgids for each container.
+	// They should be using independent sets of uids and gids if possible.
+	//
+	// MapAllSubIDs requires newuidmap(1) and newgidmap(1) with suid bit.
+	//
+	// When running in userns, MapAllSubIDs is ignored and
```

Comment:
As this is very confusing, I'm temporary closing this PR. 

Please also see https://github.com/opencontainers/runc/issues/1837